### PR TITLE
database done & map and navbar created

### PR DESCRIPTION
**Pull request by Kino-ki**

### CODE REVIEW

**Review by Halicksse — APPROVED** [review]

**Review by Lambseb — APPROVED** [review]
be careful to commit syntaxe

File: frontend/src/components/ArtworksForm.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import axios from "axios";
+import Proptypes from "prop-types";
+
+export default function ArtworksForm({ setIsUpdated }) {
+  const handleFormSubmit = (e) => {
+    e.preventDefault();
+
+    axios
+      .post(`${import.meta.env.VITE_BACKEND_URL}/api/artwork`, {
+        title: e.target[0].value,
```
Comment (ioayoub) [inline frontend/src/components/ArtworksForm.jsx:10]: Something wrong here, you are sending the same value on both lines

File: frontend/src/components/Game/Map/LocateControl.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// import * as L from "leaflet";
```
Comment (ioayoub) [inline frontend/src/components/Game/Map/LocateControl.jsx:1]: Remove this file if not used

File: frontend/tailwind.config.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+/* eslint-disable prettier/prettier */
```
Comment (ioayoub) [inline frontend/tailwind.config.js:1]: Nice try

File: frontend/tailwind.config.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ["./index.html", "./src/**/*.{js,ts,jsx,tsx}"],
+  theme: {
+    extend: {
+      colors: {
+        // eslint-disable-next-line prettier/prettier
```
Comment (ioayoub) [inline frontend/tailwind.config.js:8]: Nice try

File: frontend/tailwind.config.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+        "primary": "#684411",
+        // eslint-disable-next-line prettier/prettier
```
Comment (ioayoub) [inline frontend/tailwind.config.js:10]: 👀

**Review by ioayoub — CHANGES_REQUESTED** [review]
Comments + Remove all useless empty files

**Review by Jubdc — APPROVED** [review]

**Review by Jhanemba64 — APPROVED** [review]

**Pull request closed without merging**